db/fixtures/miq_product_features.yml

```diff
@@ -3971,6 +3971,10 @@
       :description: Edit Tags of Middleware Domains
       :feature_type: control
       :identifier: middleware_domain_tag
+    - :name: Stop Middleware Domain
+      :description: Stop Middleware Domain
+      :feature_type: admin
```

Comment:
@Fryguy shouldn't operations/features like this be also moved to the relevant provider repos?
@cfcosta fyi